### DISCUSSION TITLE
"ImageFileError: Cannot work out file type" for amsterdam sample subject

### DISCUSSION THREAD

**Discussion by dhughe13** [McIntosh-Lab/tvb-ukbb#170] [General]
Hi,

I am attempting to run the sample subject provided with the pipeline but I keep running into this error regarding one of the rawdata files. The pipeline begins to run and starts to identify which files it will be running on (the FILETYPE, PATT section), and then it shows this error:

Traceback (most recent call last):
  File "/home/hughes/tvb-ukbb/bb_pipeline_tools/bb_pipeline.py", line 156, in <module>
    main()
  File "/home/hughes/tvb-ukbb/bb_pipeline_tools/bb_pipeline.py", line 83, in main
    fileConfig = bb_file_manager(subject)
  File "/home/hughes/tvb-ukbb/bb_pipeline_tools/bb_file_manager.py", line 786, in bb_file_manager
    action(listFiles, *args)
  File "/home/hughes/tvb-ukbb/bb_pipeline_tools/bb_file_manager.py", line 315, in manage_fMRI
    epi_img = nib.load(fileName)
  File "/opt/tvb-pipeline/software/env/lib/python3.7/site-packages/nibabel/loadsave.py", line 55, in load
    raise ImageFileError(f'Cannot work out file type of "{filename}"')
nibabel.filebasedimages.ImageFileError: Cannot work out file type of "/home/hughes/TVB_data/rawdata/func/sub-0001_task-restingstate_acq-seq_bold.nii.gz"

Any ideas on how to correct this issue? I am having trouble identifying what's going wrong.

Best,
Dominique Hughes

**Comment by justinliuyuwang** [comment]
Hi Dominique! 

This may be an image header issue like [here](https://github.com/nipy/nibabel/issues/898). Could you please let me know what the output of the following is? Thank you!

`
zcat /home/hughes/TVB_data/rawdata/func/sub-0001_task-restingstate_acq-seq_bold.nii.gz | head -c 1024 > trunc.nii
`

Best,
Justin

**Comment by justinliuyuwang** [comment]
Additionally, have you [installed git-lfs](https://github.com/McIntosh-Lab/tvb-ukbb/wiki/Installing-on-a-General-HPC-System#install-git-lfs) prior to cloning, pulling from, or switching to the amsterdam branch? Could you please let me know what the output of `du -sh /home/hughes/TVB_data/rawdata/func/sub-0001_task-restingstate_acq-seq_bold.nii.gz` is?

**Reply by dhughe13** [reply]
Hi,

I had not installed git-lfs before switching to the amsterdam branch. After installing git-lfs, we re-cloned the repo again and that error has been resolved. The pipeline is running on the sample subject now.

Thank you so much for your help and your quick response!

**Reply by justinliuyuwang** [reply]
Great to hear it worked for you. No problem, anytime!

**Comment by justinliuyuwang** [comment]
Hi Dominique, in case you're looking to use the main branch, I've recently made some important changes to the pipeline for generating correct TVB inputs. Please be sure to pull the most recent version of the repository before running your data!
